README.rst

```diff
@@ -70,9 +70,12 @@ Community
 
 Rubicon is part of the `BeeWare suite`_. You can talk to the community through:
 
-* `@pybeeware on Twitter`_
+* `@pybeeware on Twitter <https://twitter.com/pybeeware>`__
 
-* The `beeware/general`_ channel on Gitter.
+* `Discord <https://discord.com/channels/836455665257021440/836455665257021443>`__
+  (`Click here for an invitation to the server <https://beeware.org/bee/chat/>`__)
```

Comment:
I would list just the invite link here, because it works regardless of whether you've already joined the server (if you have, it acts as a link to the server, and if not, it lets you join). The channel link on the other hand only works if you've already joined the server before.

```suggestion
* `Discord <https://beeware.org/bee/chat/>`__
```

Reply:
\*bump\*

Reply:
Sorry - I started looking into this, and got distracted. 

I've been trying  to verify what you're saying - if *I* follow the /bee/chat link, I still see text about accepting an invite, rather than joining the room; and if I proceed, I end up in the admins room. I'm not sure if that's because it's the first room that *I* can see, though.

I haven't been able to find any good docs or examples for what constitutes best practice here... maybe I should ask on Twitter?

Reply:
Interesting. If I open https://beeware.org/bee/chat/ in the browser (Firefox on macOS), it goes to a Discord page, which then redirects me into the Discord desktop app and opens the BeeWare #general chat. I checked, and it doesn't just use whatever channel I had open last - it specifically switches to #general.

Just to check, am I in the admin channel that it sends you to? Maybe the invite is configured to go to that specific channel, and for people that don't have access to that channel, it just sends you to the default channel instead. You could try creating an invite link for the #general channel specifically (via the channel's right-click menu) and see if that behaves any differently.

I'm not in a huge number of Discords, but in my experience everyone just posts invite links. I've never seen a non-invite channel link used for telling people what channel to go to.

Reply:
I've done some more looking around, and it looks like you're right - at least until you have a whole lot of server boosts and you can claim a discord.gg shortlink. The UX suuuuuucks, but I guess it's what we've got.